components/ActivitiesForm/ActivitiesForm.js

```diff
@@ -0,0 +1,139 @@
+import { useState } from "react";
+import initialActivities from "@/activities.json";
+import * as Styled from "./ActivitiesForm.styled";
+import NavButton from "../NavButton/NavButton";
+import { mutate } from "swr";
+
+export default function ActivitiesForm({ handleShowForm }) {
+  const [selectedEmotions, setSelectedEmotions] = useState([]);
+  const [activities, setActivities] = useState(initialActivities);
+  function handleCheckboxChange(emotion, isChecked) {
+    if (isChecked) {
+      setSelectedEmotions([...selectedEmotions, emotion]);
+    } else {
+      setSelectedEmotions(selectedEmotions.filter((e) => e !== emotion));
```

Comment:
```suggestion
      setSelectedEmotions(selectedEmotions.filter((selectedEmotion) => selectedEmotion !== emotion));
```

Reply:
usually e is short for event, so i recommend to just name it fully to make it easier to read